### ISSUE TITLE
How do I change Logistic to Softmax in YOLOv3 ? Where is yolov3 loss function?

### ISSUE THREAD

**Issue by sogou-li** [pjreddie/darknet#939]

**Comment by iraadit** [comment]
Hi @sogou-li 
Did you find how to do it?
Thanks
